controllers/consoleplugin/consoleplugin_objects.go

```diff
@@ -184,6 +185,18 @@ func buildArgs(desired *flowslatest.FlowCollectorSpec) []string {
 			args = append(args, "--loki-ca-path", helper.GetCACertPath(&desired.Loki.TLS, lokiCerts))
 		}
 	}
+
+	statusTLS := helper.LokiStatusTLS(&desired.Loki)
+	if statusTLS.Enable {
+		if statusTLS.InsecureSkipVerify {
+			args = append(args, "-loki-status-skip-tls")
+		} else {
+			args = append(args, "--loki-status-ca-path", helper.GetCACertPath(&statusTLS, lokiStatusCerts))
+			args = append(args, "--loki-status-user-cert-path", helper.GetUserCertPath(&statusTLS, lokiStatusCerts))
+			args = append(args, "--loki-status-user-key-path", helper.GetUserKeyPath(&statusTLS, lokiStatusCerts))
+		}
+	}
+
```

Comment:
can we extend unit-test to cover this newly added field ?

Reply:
https://github.com/netobserv/network-observability-operator/pull/307/commits/7d64e7c3fd3a2f9a6912dc138cc6307ea34c76bd#diff-22b7d1b615362ec58a371291942fa2bfe7290375729e4fb6b0be03f3af4ca993